src/assets/styles/colors.scss

```diff
@@ -0,0 +1,4 @@
+$colors-purple: #44318D;
+$colors-light-purple: #8162FF;
+$colors-light-gray-0: #757575;
+$colors-light-gray-1: #999;
```

Comment:
Lets follow this pattern for partials defined bu scss and name the file `_colors.scss`